faq.rst

```diff
@@ -15,21 +24,49 @@ Using 90DNS (163.172.141.219 or 45.248.48.62) or enabling GDPR protections is us
 
 ........
 
+I can't load Homebrew Launcher?
+-------------------------------
+
+When trying to access Album, or any game, hold the R button and this will launch Homebrew Launcher.
+
+........
+
+What is AutoRCM? I heard it drains the battery too?
+---------------------------------------------------
+
+.. important:: UNDER NO CIRCUMSTANCES SHOULD YOU ENABLE AUTORCM ON AN IPATCHED SWITCH. THIS WILL CAUSE YOU TO BRICK YOUR SWITCH AS YOU CANNOT GET OUT OF IT!
+
+AutoRCM essentially corrupts part of your boot sector in order to force your Switch to always boot into RCM mode. This is incredibly handy for two reason:
+
+* You no longer need a jig to access RCM 
+* It stops you from accidently burning fuses
+
+AutoRCM will only drain your battery if you leave your Switch in RCM mode, which is a black screen but your Switch is still "on". To stop this, make sure you shut your Switch down properly, or hold power for 12 seconds.
+
+........
+
+FAT32 or ExFAT?
+-----------------------------
+
+Unfortunately ExFAT on the Switch, specifically with Custom Firmware, has terrible drivers. This can easy corruption and force you to start again if this happens.
+
+We highly recommend you stay with FAT32 as it almost never corrupts, and there is no draw backs since the Switch automatically splits it's games on FAT32, and Hekate splits NAND back ups for you.
+
+In order to format an SD Card on Windows above 32GB, use GuiFormat.

```

Comment:
Change `GuiFormat` to <p><code>\`GUIFormat \<http://www.ridgecrop.demon.co.uk/index.htm?guiformat.htm>`_</code></p> to link to it.